docs/storage-policies.md

```diff
@@ -100,6 +104,7 @@ To determine which storage policy is assigned to a user, call
 [`storagePolicies.getAssignmentForTarget(userID, callback)`][getAssignmentForTarget]
 with the ID of the user.
 
+<!-- sample get_storage_policy_assignments -->
```

Comment:
API WG has been promoting a pattern where related objects (e.g. the assignment for a user) are accessed via a query parameter on the related collection resource (`GET /storage_policy_assignments?user_id=ID`) rather than as a subresource on the individual resource (e.g. `GET /users/ID/storage_policy_assignment`, which does not exist).  This could be confusing to document if the collection resource should also be used more generally to get all of the elements in the collection (e.g. just `GET /storage_policy_assignments`, which is not used today for this endpoint) — any idea how we should tag those going forward, since theoretically they correspond to different logical operations?

Reply:
Interesting. I'd prefer the alternative actually. I think of endpoints as unix commands: do one thing and do it well. 

As for this issue, ideally, I'd like the ref docs to just document all params for an endpoint and then have good guides, recipes, etc that showcase each use case for each endpoint. Yet, we can explore different ways to highlight usage of endpoints as we progress. This is already a thing we have to explore with some of the existing endpoints anyway.

As for the samples, I'd love if we can at least expose one method per SDK that matches closely to each endpoint. We can then handle the variations and clever functions in a different way in variation samples.